index.js

```diff
@@ -67,10 +68,113 @@ class BotiumConnectorGoogleAssistant {
   UserSays ({ messageText }) {
     debug('UserSays called')
     debug(`Request: ${messageText}`)
+    const getCards = (response) => {
+      let result = []
+      if (response.cards) {
+        result = result.concat(response.cards.map(c => {
+          return {
+            text: c.title || c.subtitle || c.text,
```

Comment:
Bei den cards können text, subtext, content gesetzt werden - muss nicht alles in das "text" feld fließen